UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -359,6 +359,16 @@ public void ServerSetJobTitle(string newJobTitle)
 		SyncJobTitle(jobTitle, newJobTitle);
 	}
 
+	public Clearance GetHighestClearance()
+	{
+		var highestClearance = Clearance.MaintTunnels;
+		foreach (var variabClearance in GetClearance)
+		{
+			if (variabClearance > highestClearance) highestClearance = variabClearance;
```

Comment:
this assumes the order at which they are declared matters, it actually doesn't.

Reply:
left as a todo for later

Reply:
Returns the highest clearance you have, right now it won't work well with security but for other roles I think it will work fine